lib/ansible/modules/monitoring/icinga2_host.py

```diff
@@ -248,7 +252,7 @@ def main():
     variables = module.params["variables"]
 
     try:
-        icinga = icinga2_api()
+        icinga = icinga2_api(use_proxy=module.params["use_proxy"])
```

Comment:
Why not pass `module` instead (and remove that `icinga.module = module`) in the next line?

Reply:
@felixfontein Could you please verify current version?